src/pages/beer/[id].tsx

```diff
@@ -0,0 +1,137 @@
+        }
+      >
+        <div className={'flex w-full justify-between'}>
+          <div className={'flex flex-col'}>
+            <Heading1>{beer.name}</Heading1>
+            <Paragraph>{beer.tagline}</Paragraph>
+          </div>
+          <div className={'relative h-48 w-24'}>
+            <Image
+              src={beer.image_url}
+              alt={beer.name}
+              layout={'fill'}
+              objectFit={'contain'}
+              objectPosition={'top'}
+            />
+          </div>
+        </div>
+        <div className="flex flex-col justify-between mt-4 lg:flex-row lg:gap-5">
+          <div className="flex flex-col w-full mb-2 lg:mb-0 lg:w-3/4">
+            <p className="mb-2 text-3xl font-light">{beer.description}</p>
+            <p className="text-3xl font-semibold">Ingredients:</p>
+            <p className="text-2xl">Malt</p>
+            <ul>
+              {beer.ingredients.malt.map((ingredient) =>
+                <li key={ingredient.name}>
+                  {ingredient.name} - {ingredient.amount.value}{' '}
+                  {ingredient.amount.unit}
+                </li>
+              )}{' '}
+            </ul>
+            <p className="text-2xl">Hops</p>
+            <ul>
+              {beer.ingredients.hops.map((ingredient) =>
+                <li key={ingredient.name}>
+                  {ingredient.name}- {ingredient.amount.value}{' '}
+                  {ingredient.amount.unit}
+                </li>
+              )}
+            </ul>
+          </div>
+          <ul className={'flex w-full flex-col lg:w-1/4'}>
+            <li>
+              <p>
+                <span className={'font-semibold'}>Brewers Tips:</span>{' '}
+                {beer.brewers_tips}
+              </p>{' '}
+            </li>
+            <li>
+              <p>
+                <span className={'font-semibold'}>First Brewed:</span>{' '}
+                {beer.first_brewed}
+              </p>{' '}
+            </li>
+            <li>
+              <p>
+                <span className={'font-semibold'}>Contributed By:</span>{' '}
+                {beer.contributed_by}
+              </p>{' '}
+            </li>
+            <li>
+              <p>
+                <span className={'font-semibold'}>Attenuation Level:</span>{' '}
+                {beer.attenuation_level}
+              </p>{' '}
+            </li>
+            <li>
+              <p>
+                <span className={'font-semibold'}>Boil Volume:</span>{' '}
+                {beer.boil_volume.value} {beer.boil_volume.unit}
+              </p>{' '}
+            </li>
+            {' '}
+            <li>
+              <p>
+                <span className={'font-semibold'}>Fermentation:</span>{' '}
+                {beer.method.fermentation.temp.value}{' '}
+                {beer.method.fermentation.temp.unit}
+              </p>{' '}
+            </li>
+          </ul>
+        </div>
+      </main>
+    </UI>
+  )
+}
+
+export default Beer
```

Comment:
starajmy sie unikać domyslnych eksportów https://javascript.works-hub.com/learn/Why-we-have-banned-default-exports-in-JavaScript-and-you-should-do-the-same

w konfiguracji eslinta ta flaga była zakomentowana, pewnie przypadkiem tak zrobiłem ale było to tymczasowo :)